container/raw/handler.go

```diff
@@ -336,6 +336,10 @@ func (self *rawContainerHandler) GetStats() (*info.ContainerStats, error) {
 	return stats, nil
 }
 
+func (self *rawContainerHandler) GetProcesses() ([]int, error) {
```

Comment:
leftover from before I realized that we already had a ListProcesses() :) removed.